cfg/conf.sample.php

```diff
@@ -18,6 +18,11 @@
 ; preselect the discussion feature, defaults to false
 opendiscussion = false
 
+; enable or disable the diplay of dates & times in the comments, defaults to true
+; note that internally the creation time will still get tracked in order to sort
+; the comments by creation time, but you can choose not to display them
```

Comment:
```suggestion
; Note that internally the creation time will still get tracked in order to sort
; the comments by creation time, but you can choose not to display them.
```

Let's make it a proper sentence.